operations/node_management/Rebuild_NCNs/Prepare_Storage_Nodes.md

```diff
@@ -45,11 +45,182 @@ If rebuilding `ncn-s001`, it is critical that the `storage-ceph-cloudinit.sh` ha
 
 ## Procedure
 
+Upload ceph container images into nexus.
+
+1. (`ncn-s#`) Copy and paste the below script into `/srv/cray/scripts/common/upload_ceph_images_to_nexus.sh` **on the node being rebuilt**.
+
+    ```bash
+    #!/bin/bash
```

Comment:
This should move into a script file, and then the `chmod` step can go away as well as this one.